evals/registry/eval_sets/audio.yaml

```diff
@@ -3,7 +3,7 @@ audio:
     - audio-*.test
     - audio-boolq
     - audio-ser
-    - audio-sqa
+    - audio-sqa-*.validation.v1
```

Comment:
Is there a subset that we want to add to our `audio-core` as well?

Also, it seems like the `audio` evalset has a few elements more than `transcript`. Is that intended? I was gonna say it makes sense to not include SER, but even for SER it might make sense to have the text benchmark for it.

Reply:
maybe. These are not large datasets so eval is fairly fast, I think we probably should start adding SQA to the core dataset if it makes sense to you.

Currently some of the datasets only have audio versions of the inputs, so it's not straightforward to add text-only variants. I think that's only sWUGGY and sBLIMP at the moment, but let me know if you see other cases.

Reply:
Makes sense.

I guess whenever we find a good eval that we want to be in every run then we add it to `-core`.

For example we thought BoolQ was good, but then it turned out it added very little insight. Datasets that are saturated fall into this category very often.

So if we do very poorly on a dataset (e.g. sWUGGY and sBLIMP?), it might be a good candidate to add (we can decide to remove it later).